src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -305,6 +306,9 @@ public void reset() {
 
     myEnableEmbeddedBrowsersCheckBox.setSelected(settings.isEnableEmbeddedBrowsers());
 
+    // TODO(helinx): Remove this once JxBrowser is compatible with MacOS 11.
+    myEnableEmbeddedBrowsersCheckBox.setEnabled(!SystemInfo.isMacOSBigSur);
```

Comment:
Perhaps also (conditionally on if we're on big sur):

```
myEnableEmbeddedBrowsersCheckBox.setToolTipText("This option not available...");
```

Reply:
Oh I didn't realize you could set the tool tip here! Thanks